src/app/choose-template/page.tsx

```diff
@@ -0,0 +1,42 @@
+'use client' // we need this to be a client component because nhsuk-react-components uses client-only react features
+import { Radios, Button, ErrorSummary } from 'nhsuk-react-components';
+import { useFormState } from 'react-dom';
+import { chooseTemplateServerAction, FormState } from '../../utils/choose-template-server-action';
```

Comment:
I feel like the server actions should sit along side the page rather than in `utils`, especially since they are probably not re-usable and have generically named types, i.e. `FormState`.

Reply:
updated